egs/babel_multilang/s5/local/nnet3/run_tdnn_multilingual.sh

```diff
@@ -204,9 +204,12 @@ EOF
     echo " output-layer name=output-${lang_index} dim=$num_targets max-change=1.5"
   done >> $dir/configs/network.xconfig
 
+  cat <<EOF > $dir/configs/edits.config
+  rename-node old-name=output-0 new-name=output
+EOF
   steps/nnet3/xconfig_to_configs.py --xconfig-file $dir/configs/network.xconfig \
     --config-dir $dir/configs/ \
-    --nnet-edits="rename-node old-name=output-0 new-name=output"
+    --nnet-edits=$dir/configs/edits.config
```

Comment:
nnet-edits expects a string, not a filename.  also you are already supplying --config-dir as an arg to the script and I see that elsewhere you are treating edits.config as a standard part of that config directory, so there seems to be some disagreement about whether scripts are supposed to pick that up automatically, or whether it needs to be supplied explicitly.
If you are intending to change how that option works you should rename it and chang the documentation.